tools/misc/rawparser.py

```diff
@@ -49,13 +49,11 @@
 startOfFrame = 0
 metadataLength = 128
 logImage = False
-raw_img_dir = '\\sample_raw\\'
-fileName = 'frames202309111819'
-rawFileType = '.RAW'
-binFileType = '.bin'
 pngFileType = '.png'
 plyFileType = '.ply'
 
+rawParserVersion = "1.0.0"
+TOFEvalVersion = "5.0.0"
```

Comment:
You can get the SDK version directly using this:
https://github.com/analogdevicesinc/ToF/blob/main/bindings/python/aditofpython.cpp#L859
So you don't need to update this variable manually for each version increase.

Reply:
As of now, rawparser is not dependent on the python bindings. I'll work on it.

Reply:
Oh, I didn't realize that. Then I guess it's fine.